third_party/js.bzl

```diff
@@ -166,6 +166,10 @@ def tensorboard_js_workspace():
                 "http://mirror.tensorflow.org/raw.githubusercontent.com/google/material-design-icons/b3f05bfbf4329a5b63f50a720f867c2bac163f98/src/alert/warning/materialicons/24px.svg",
                 "https://raw.githubusercontent.com/google/material-design-icons/b3f05bfbf4329a5b63f50a720f867c2bac163f98/src/alert/warning/materialicons/24px.svg",
             ],
+            "c1f1e1f555e04b9444d6af02d8db242c4dc3d0bb34467f11a6b7c538dfd2f584": [
+                "http://mirror.tensorflow.org/raw.githubusercontent.com/google/material-design-icons/4.0.0/src/social/notifications_none/materialicons/24px.svg",
```

Comment:
Is there a specific you are taking it from 4.0.0 tag instead of a commit like other items?

Reply:
Thanks for the comment and offline discussion. Replaced version tag with commit number